test/fixture.js

```diff
@@ -1,556 +0,0 @@
-'use strict';
```

Comment:
@domoritz  This file demonstrates that `git mv` is useless since I already did `git mv` in [this commit](https://github.com/vega/compass/commit/13c17f9fd8c19440e8e68e59b060da63905b7198), but later changes still override that and make the history discontinuous!  







https://github.com/vega/compass/pull/143/files#diff-eff32a34aae02361ca776f259779ff1eL1